src/server/api/routers/commission.ts

```
@@ -125,42 +125,40 @@ export const commission_router = createTRPCRouter({
             await update_index('commissions', {
                 objectID: input.id,
-                title: updated_commission[0]!.title,
-                price: format_to_currency(updated_commission[0]!.price / 100),
-                description: updated_commission[0]!.description,
-                featured_image: get_ut_url(updated_commission[0]!.images[0]!.ut_key),
-                slug: updated_commission[0]!.slug,
+                title: updated_data.title,
+                price: updated_data.price
+                    ? format_to_currency(updated_data.price / 100)
+                    : undefined,
+                description: updated_data.description!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)